main.py

```diff
@@ -11,6 +12,8 @@ def welcome():
     username= request.form['username']
     password= request.form['password']
     verify_password = request.form['verify_password']
+    email = request.form['email']
+    pattern = r"[A-z0-9]+@[A-z0-9]+\.[A-z0-9.]+"
```

Comment:
Extract the email text field from the form, and set a regex pattern to match a valid email. Basically this pattern means, "any character or number at any character or number dot any character or number"